src/commands/project/deploy/resume.ts

```diff
@@ -37,6 +36,7 @@ export default class DeployMetadataResume extends SfCommand<DeployResultJson> {
     'job-id': Flags.salesforceId({
       char: 'i',
       startsWith: '0Af',
+      length: 'both',
```

Comment:
Added this so linting would display "0 problems" rather than all the warnings that suggested using everything offered by the `salesforceId` flag.